lib/block-supports/spacing.php

```diff
@@ -35,8 +35,12 @@ function gutenberg_register_spacing_support( $block_type ) {
  * @return array Block spacing CSS classes and inline styles.
  */
 function gutenberg_apply_spacing_support( $block_type, $block_attributes ) {
-	$has_padding_support = gutenberg_has_spacing_feature_support( $block_type, 'padding' );
-	$has_margin_support  = gutenberg_has_spacing_feature_support( $block_type, 'margin' );
```

Comment:
Removed `gutenberg_has_spacing_feature_support` as it was just a wrapper for `gutenberg_block_has_support`. Unless the support config is changed to allow all spacing support with a single boolean it isn't needed.